quickstart-node/src/services/jwt.service.ts

```diff
@@ -0,0 +1,52 @@
+import { EnvironmentVariablesService, SecretsService } from '@services/monday-code';
+import jwt, { JsonWebTokenError, JwtPayload, TokenExpiredError } from 'jsonwebtoken';
+import { InternalServerError, Unauthorized } from 'http-errors';
+import axios from 'axios';
+import { EnvironmentsKeys, SecretsKeys } from '@shared/monday-auth-consts';
+
+export class JWTService {
+  private constructor() {}
+
+  static createJWTToken(payload: object): string {
```

Comment:
Would we need to create?

Reply:
nope, but why is it also on the python example too? any specific reason?